stix2/environment.py

```diff
@@ -152,3 +152,17 @@ def add(self, *args, **kwargs):
     def parse(self, *args, **kwargs):
         return _parse(*args, **kwargs)
     parse.__doc__ = _parse.__doc__
+
+    def created_by(self, obj):
```

Comment:
I'm trying to think about the best name for this function. I'm leaning towards `creator_of()` or `get_creator_of()`, since the idea behind the Workbench API layer is that all of the environment methods will be exposed as module-level functions on the "implicit `Environment`" of the Workbench. The `get_` prefix is (to me) an indicator that the operation is potentially expensive (depending on the type of DataSource), but I don't know if we want a ton of `get_*` functions in the Workbench API.

Reply:
I thought about this, too. My reason for going with this was an idea that in the workbench layer you could do `myobj.created_by()` to get the creator object. But since that would be a wrapper around this function I guess it doesn't need to be named the same (i.e. `myobj.created_by()` could call `creator_of(myobj)` under the hood).

I'll rename it.